stout/static-thread-pool.cc

```diff
@@ -69,6 +73,11 @@ StaticThreadPool::StaticThreadPool()
 
               STOUT_EVENTUALS_LOG(1) << "Resuming '" << waiter->name() << "'";
 
+              STOUT_EVENTUALS_LOG(1) << "Thread " << core << "(id="
+                                     << std::this_thread::get_id()
+                                     << ") is running on core "
+                                     << GetRunningCore();
```

Comment:
Let's move this up to just after we set the affinity. No need to print it on the log every time! Or if you're thinking that this will be useful for debugging if affinity isn't working then let's increase the log level to 2 or even 3 (not 1). Also see the code suggestion where I've added a space before the `(` so that the core number is easier to read.

```suggestion
              STOUT_EVENTUALS_LOG(1) << "Thread " << core << " (id="
                                     << std::this_thread::get_id()
                                     << ") is running on core "
                                     << GetRunningCore();
```